### ISSUE TITLE
Bug Report (Empty column tagged as Unsupported)

### ISSUE THREAD

**Issue by joshua-lagasca-novare** [ydataai/pandas-profiling#1104]
### Current Behaviour

An empty non-object datatype column must be tagged with an 'Empty' alert.

### Expected Behaviour

An empty non-object datatype column returns tagged with an 'Unsupported' alert.

### Data Description

An empty non-object datatype column.

### Code that reproduces the bug

```Python
import pandas as pd
from pandas_profiling import ProfileReport

df = pd.DataFrame(columns=['a','b']).astype(int)
report = ProfileReport(df, title="bug report", minimal=True)
report.to_file("report.html")
```

### pandas-profiling version

v3.3.0

### Dependencies

```Text
pandas==1.3.5
```

### OS

Ubuntu 20.04.5 LTS x86_64

### Checklist

- [X] There is not yet another bug report for this issue in the [issue tracker](https://github.com/ydataai/pandas-profiling/issues)
- [X] The problem is reproducible from this bug report. [This guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) can help to craft a minimal bug report.
- [X] The issue has not been resolved by the entries listed under [Common Issues](https://pandas-profiling.ydata.ai/docs/master/pages/support_contrib/common_issues.html).

**Comment by Mathanraj-Sharma** [comment]
Shall I take this ticket and work on this?

@fabclmnt

**Comment by fabclmnt** [comment]
@Mathanraj-Sharma feel free to submit a PR with a proposed solution!

**Comment by alexbarros** [comment]
hi @Mathanraj-Sharma, are you still working on this one?

**Comment by alexbarros** [comment]
@Sohaib90 you can go for it

**Comment by Sohaib90** [comment]
@alexbarros I think it has already been worked on? Attaching the "Expected behavior" screenshot for reference:

![image](https://user-images.githubusercontent.com/43241257/212885181-d9512988-2121-4786-905b-e07c2e9a15e6.png)

I think you can close this issue?

**Comment by alexbarros** [comment]
@Sohaib90 yeah, I believe it is a non-issue in the end, but I believe we should raise an exception in case we try to generate a report of an empty data frame so the error became more obvious faster.

**Comment by Sohaib90** [comment]
Alright. I will work on that then :)

**Comment by Sohaib90** [comment]
@alexbarros I have created a PR for the issue. Maybe you can look into it :)
